### PR TITLE
Try listr

### LINKED ISSUE #217: Move from ORA spinner to LISTR
https://github.com/SamVerschueren/listr

So we can show progress cleanly.  This cannot be done with ora
https://github.com/sindresorhus/ora/issues/31

### PR DESCRIPTION
**Pull request by GantMan**
FIX SPINNERS!

Destroy tons of tests 👎 

All tests were manual.  This was one of the major lynchpins of Solidarity 3.0 coming out.

Closes #217

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ yarn.lock
 config.yml
 pretty.yml
 render*.gif
+package-lock.json
```
Comment (GantMan) [inline .npmignore]: this is for you Jamon!

File: __tests__/command_helpers/checkRequirement.ts [hunk 2]
```diff
@@ -120,43 +118,48 @@ describe('checkRequirement', () => {
...
-      const result = await checkRequirement(rule, context)
-      expect(result).toEqual(["'$yarn' environment variable not found"])
-    })
+    //   const rule = toPairs({
+    //     YARN: [{ rule: 'env', variable: 'yarn' }],
+    //   })[0]
+    //   const listrTask = await checkRequirement(rule, context)
+    //   const result = await listrTask.storedInit[0].task()
+    //   expect(result).toEqual(undefined)
+    // })
```
Comment (jamonholmgren) [inline __tests__/command_helpers/checkRequirement.ts]: Can you add a note as to why this is commented out?

File: __tests__/command_helpers/checkRequirement.ts [hunk 3]
```diff
@@ -168,70 +171,77 @@ describe('checkRequirement', () => {
...
-      const result = await checkRequirement(rule, context)
-      expect(result).toEqual(["'yarn' file not found"])
-    })
+    //   const rule = toPairs({
+    //     YARN: [{ rule: 'file', location: 'yarn' }],
+    //   })[0]
+    //   const listrTask = await checkRequirement(rule, context)
+    //   const result = await listrTask.storedInit[0].task()
+    //   expect(result).toEqual("'yarn' file not found")
+    // })
```
Comment (jamonholmgren) [inline __tests__/command_helpers/checkRequirement.ts]: Same here (reason for commented out).

File: package.json [hunk 4]
```diff
@@ -37,44 +37,45 @@
   "license": "MIT",
   "dependencies": {
     "callsite": "^1.0.0",
-    "envinfo": "^5.12.0",
+    "envinfo": "6.0.1",
     "gluegun": "2.0.0-beta.11",
```
Comment (jamonholmgren) [inline package.json]: 😞

**Pull request merged**